x-pack/plugins/kubernetes_security/public/components/tree_view_container/breadcrumb/index.tsx

```diff
@@ -50,28 +57,46 @@ export const Breadcrumb = ({ treeNavSelection, onSelect }: BreadcrumbDeps) => {
   const renderBreadcrumbLink = useCallback(
     (
       collectionType: KubernetesCollection,
-      icon: JSX.Element,
+      treeViewIconProps: TreeViewIconProps,
       isBolded: boolean,
       hasRightArrow: boolean = true
     ) => {
+      const clusterLevel =
+        collectionType === KubernetesTreeViewLevels.clusterId
+          ? 'Cluster'
+          : startCase(collectionType);
       const content =
-        collectionType === 'clusterId'
+        collectionType === KubernetesTreeViewLevels.clusterId
           ? treeNavSelection.clusterName || treeNavSelection.clusterId
           : treeNavSelection[collectionType];
 
+      const tooltip = `${clusterLevel}: ${content}`;
+      const showBreadcrumbText = showBreadcrumbDisplayText(treeNavSelection, collectionType);
+      const { type: iconType, euiVarColor } = treeViewIconProps;
+
       return (
         <>
           {hasRightArrow && <EuiIcon css={styles.breadcrumbRightIcon} type="arrowRight" size="s" />}
-          {icon}
-          <EuiToolTip content={content}>
-            <EuiButtonEmpty
-              css={isBolded ? styles.breadcrumbButtonBold : styles.breadcrumbButton}
-              color="text"
+          <EuiToolTip content={tooltip}>
+            <EuiButtonIcon
+              data-test-subj={`kubernetesSecurityBreadcrumbIcon-${collectionType}`}
+              iconType={iconType}
+              style={{ color: euiVars[euiVarColor] }}
+              aria-label={`Click ${clusterLevel} breadcrumb`}
               onClick={() => onBreadCrumbClick(collectionType)}
```

Comment:
that was missed in the first time, can you add this to a callback?

in a way that we can use like this: onClick={onBreadCrumbClickCallback}